.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,10 +1,5 @@
 ---
-name: Bug report 🐞
-about: Create a report to help us improve
-title: ''
-labels: bug
-assignees: ''
-
+about: Create a bug report to help us improve
```

Comment:
I kept this to preserve this text:
![image](https://user-images.githubusercontent.com/9084735/119687200-03117b00-be15-11eb-97ab-705594f94f83.png)

Not sure it will work

Reply:
Now these props (about) look duplicated.. We really need that on both files now?

Reply:
I kept the text slightly different to check which one is being used. There is no other way to really test this 🙁

If we find it's not being used, we can remove it

Reply:
But you did remove the name..and it is shown, I guess.
So should be same.

Reply:
it is shown? no that is not a real screenshot. My guess is the name will just say "Issue".

Again, there is no way to test this without actually merging it.